# This service defines a look-at-position action for a CommU robot. This is done by providing an (`look_x`, `look_y`, `look_z`) position the
# robot should look at.

# The x-axis is horizontal, increases towards left from the robot and is 0 between its eyes. In millimeters.
int8 look_x

# The y-axis is vertical, increases towards the head of the robot and is 0 at the bottom of its body. In millimeters.
int8 look_y

# The z-axis is horizontal, increases in front of the robot with 0 being at the center of rotation of the head yaw. In millimeters.
int8 look_z

---

#Whether the look operation was received by the CommU.
bool success